src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -415,9 +415,13 @@ def reset(self):
 
     async def start(self, dev_ctrl, node_id: int, endpoint: int, fabric_filtered: bool = False, min_interval_sec: int = 0, max_interval_sec: int = 5, keepSubscriptions: bool = True) -> Any:
         """This starts a subscription for attributes on the specified node_id and endpoint. The cluster is specified when the class instance is created."""
-        attributes = [(endpoint, self._expected_cluster)]
+
         if self._expected_attribute is not None:
-            attributes = [(endpoint, self._expected_attribute)]
+            attributes: list[tuple[int, Union[ClusterObjects.Cluster, ClusterObjects.ClusterAttributeDescriptor]]] = [
```

Comment:
how about using type alias for this type: `list[tuple[int, Union[ClusterObjects.Cluster, ClusterObjects.ClusterAttributeDescriptor]]]` in order to improve readability?

e.g.:

```python
from typing import TypeAlias

UserRecord: TypeAlias = dict[str, str | int]

def process_user_data(data: UserRecord):
    print(data["name"])
```